<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockSimulator</class>
 <widget class="QDialog" name="StockSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="investedLabel">
     <property name="text">
      <string>Invested: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="balanceLabel">
     <property name="text">
      <string>Balance:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableViewStocks"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter Purchase Quantity:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditQuantity"/>
   </item>
   <item>
    <widget class="QPushButton" name="buyStock">
     <property name="text">
      <string>Buy </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sellStock">
     <property name="text">
      <string>Sell</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
